extensions/amp-social-share/1.0/social-share-config.js

```diff
@@ -48,26 +48,38 @@ const BUILTINS = {
       'text': 'TITLE',
       'url': 'CANONICAL_URL',
     }),
+    'canCustomize': true,
+    'defaultColor': 'ffffff',
+    'defaultBackgroundColor': '1da1f2',
   },
   'facebook': {
     'shareEndpoint': 'https://www.facebook.com/dialog/share',
     'defaultParams': dict({
       'href': 'CANONICAL_URL',
     }),
+    'canCustomize': false,
```

Comment:
Let's go ahead and set it to `false` for all branded icons. We can set back to `true` if we learn anything different later. But this is a more likely scenario right now, imho.

Reply:
Sounds good, thanks!!

Reply:
I think this is always `true` now and therefore removable?

Reply:
Thanks, in the latest version, we updated the company related one to `false`.  This way the user cannot change color or provide children for these `types`.  We can change it if we hear back differently from p counsel.